modules/govuk_python/manifests/init.pp

```diff
@@ -9,5 +9,5 @@
 
   Class['python::install'] -> Package <| provider == 'pip' and ensure != absent |>
 
-  ensure_packages(['python3', 'python3-dev'])
+  ensure_packages(['python3.5', 'python3.5-dev'])
```

Comment:
I think it's safest to keep installing `python3-dev`, as we're not managing this at a granular level, so we have no real idea of what uses Python 3.4.

Reply:
That sounds sensible, one thing that also hadn't occurred to me is that there are probably other packages which depend on `python3` and so we're getting it anyway - but better to be explicit.

Reply:
Thanks both! I've updated this now in https://github.com/alphagov/govuk-puppet/pull/10191/commits/76181452c6a73d71f171f81ab64a70bffaf95ba6